README.md

```diff
@@ -1,16 +1,33 @@
 # mbed-client-cli
 
-This is the Command Line Library for a CLI application. This library provides methods for:
+This is the Command Line Library for a CLI application. It uses only ansi C features so it is portable and works in mbed-os-3, mbed-os-5, linux and windows.
+
+## Features
+
+Library provides features such:
 
 * Adding commands to the interpreter.
 * Deleting commands from the interpreter.
 * Executing commands.
 * Adding command aliases to the interpreter.
 * Searching command arguments.
+* implements several VT100/VT220 features, e.g.
+  * move cursor left/right (or skipping word by pressing alt+up/right)
```

Comment:
for mbed-os-example binary, probably some features aren't used. like:
skipping words by `ctrl+left/right` is possible but not `alt+up/right`
CTRL+W to remove previous word is not working

I will check it later
actually, these commands wasn't added into `cmd_init()`

Reply:
did you extract mbed-os5 example out of repository and update mbed-client-cli.lib to point to this commit before compiling?

Reply:
maybe you met this: 
https://github.com/ARMmbed/mbed-cli/issues/407

Reply:
hmm - when using linux example with nurses - it alt+left/right works fine, but without ncurses it prints:
```
/>asd asdasd^[b^[f
```

Maybe I should add support for that code as well..

Reply:
actually it probably related how stdin is configured in linux example. Should try with mbed device instead..

Reply:
yes, mbed binary works, but not linux example